roxie/ccd/ccdserver.cpp

```diff
@@ -1203,11 +1206,18 @@ class CRoxieServerActivity : public CInterface, implements IRoxieServerActivity,
                         dependencies.item(idx).stopSink(dependencyIndexes.item(idx));
                 }
                 if (input)
-                    input->stop(aborting);
+                    input->stop();
             }
         }
     }
 
+    virtual void abort()
+    {
+        aborted = true;
+        // MORE - should abort dependencies here?
+        stop();
```

Comment:
I still want to stop them... calling abort first means that the sink at least knows not to do any extra work.